app/assets/stylesheets/layout/_layout.scss

```diff
@@ -45,6 +45,17 @@ body {
       margin-bottom: 40px;
     }
   }
+
+}
+
+.registrations .box, .sessions .box{
+  width: 40%!important;
```

Comment:
! should be preceeded by a space, and should not be followed by a space<br>!important should not be used<br>Properties should be ordered margin, width